src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -43,8 +43,8 @@ public List<Order> getByUser(User user) {
                                        + "LEFT JOIN FETCH ms.movie "
                                        + "LEFT JOIN FETCH ms.cinemaHall "
                                        + "LEFT JOIN FETCH o.user "
-                                       + "WHERE o.user.id = :userId", Order.class)
-                    .setParameter("userId", user.getId())
+                                       + "WHERE o.user.id = :user", Order.class)
+                    .setParameter("user", user.getId())

```

Comment:
I think here @maxline meant that you can reference o.user instead of o.user.id and pass user as a parameter instead of user.getId()